CONTRIBUTING.rst

```diff
@@ -132,3 +132,36 @@ To bring in updated upstream modules.  Update `gilt.yml` and execute the followi
 
 .. _`Ansible Goss`: https://github.com/indusbox/goss-ansible
 .. _`Gilt`: https://gilt.readthedocs.io
+
+Documentation
+=============
+
+Working with InterSphinx
```

Comment:
Why are we adding a generic example of using InterSphinx to the molecule docs?

Reply:
Well, we'll want contributors to work with InterSphinx and I didn't see such a step-by-step on how to get the links out of the inventory listing and how to read the link syntax (so I wouldn't think this is *that* generic), so I put that here. Sphinx can be intimidating to work with and I'd rather have the 'this is all you need to know' docs here locally in Molecule. That's my two cents!

Reply:
That's true. It's quite hard to find a comprehensive guide on how to use intersphinx. Sometimes I have to google for it but I only find mentions of half-baked commands here and there.
So it's probably reasonable to keep it here until sphinx starts having their own instructions...

Reply:
Okay. If I find a better solution, I'll propose it in a future PR.